### ISSUE TITLE
Improve readability

### ISSUE THREAD

**Issue by jvendrell** [MarsBased/marsbased-website#56]
We would like to improve the readability of our website, especially the content pages such as blog, work or playbook.

We would like to follow the approach of the Buffer blog:

https://blog.bufferapp.com/facebook-change-news-feed-focus-friends-family

It can't be done exactly because the typography is different. However, we can try to do something similar (check screenshots). We need to do this in a separate branch because it has a deep implication in all the pages.

**Before:**

<img width="1440" alt="before" src="https://cloud.githubusercontent.com/assets/7082300/22370846/eb058f92-e493-11e6-9c8d-650e1a89a66b.png">

**After:**

<img width="1440" alt="after" src="https://cloud.githubusercontent.com/assets/7082300/22370849/edb27e76-e493-11e6-90d2-b36295ab80fa.png">

**Comment by javierartero** [comment]
https://github.com/MarsBased/marsbased-website/commits/feature/improve-readability

**Comment by alexrodba** [comment]
@jvendrell @javierartero I don't quite fancy the colour of the text. Looks like everything has been selected an bolded (negrita).

Line height is almost ok. Try 1.75em. Looks better to me, doesn't it? We can leave this for a dedicated session any other day :)

**Comment by jvendrell** [comment]
It seems a lot bolder because the original font was very light. But I think it's a visual effect when comparing the original with the new. Most websites have a very similar font-weight. SourceSans Pro is a very common font and the font-weight in Artero's branch is the regular one.

If you spend some time reading with the new font-weight, you'll get used to it quickly.

In any case, I don't like the font color either. I think we can improve it a bit more.

I've checked the examples of this font on Google Fonts page and I've created another setup:

![screencapture-marsbased-blog-2017-02-01-looking-back-to-2016-year-consolidation-marsbased-1487065446830](https://cloud.githubusercontent.com/assets/7082300/22923741/fb4fc7d8-f2a2-11e6-9cac-5b357328c8f4.png)

(open the screenshot in your computer, at full screen, like if you were viewing the website)

This is what I've changed:

```
p
font-size: 17px;
line-height: 1.8em;
margin-bottom: 2.5rem;
color: #3d3e44;
font-weight: 400;

.main-title, .article-content h1, .article-header .title
line-height: 1.5;

h1, .h1, h2, .h2, h3, .h3, hgroup h2
margin-top: 40px;
margin-bottom: 20px;

.image-oversized+em, .article-content img+em
font-size: 1.7rem;
```

In other words:

- I've changed the paragraphs color from gray to almost black.
- I've increased the paragraphs font-weight, font-size and line-height.
- I've increased margins between paragraphs.
- I've increased margins between titles and paragraphs.
- I've increased the blog title's line-height.
- I've increased the image caption's font-size.

What do you think?
